edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -0,0 +1,185 @@
+edx = edx || {};
+
+(function(Backbone, $) {
+    'use strict';
+
+    var viewHelper, examStatusReadableFormat;
+
+    edx.courseware = edx.courseware || {};
+    edx.courseware.proctored_exam = edx.courseware.proctored_exam || {};
+
+    examStatusReadableFormat = {
```

Comment:
since this is learner facing I'm thinking we could cut down the level of detail and just render more human obvious states.  E.x. not started, started, submitted, rejected, verified, error

Reply:
That makes sense, will update